frontend/src/components/pages/connect/Overview.tsx

```diff
@@ -72,148 +70,127 @@ class TabClusters extends Component {
+                                        <span style={mr05}>{errIcon}</span>
                                         {r.clusterName}
-                                    </span>
+                                    </Tooltip>
                                 );
-                            },
-                            sorter: sortField('clusterName'),
-                            defaultSortOrder: 'ascend'
-                        },
-                        {
-                            dataIndex: 'clusterAddress',
-                            title: 'Version',
-                            render: (_, r) => r.clusterInfo.version,
-                            sorter: sortField('clusterAddress'),
-                            filterType: { type: 'enum' }
-                        },
-                        {
-                            dataIndex: 'connectors',
-                            width: 150,
-                            title: 'Connectors',
-                            render: (_, r) => <ConnectorsColumn observable={r} />
+                            }
+
+                            return (
+                                <span className="hoverLink" style={{display: 'inline-block', width: '100%'}} onClick={() => appGlobal.history.push(`/connect-clusters/${encodeURIComponent(r.clusterName)}`)}>
+                                        {r.clusterName}
+                                    </span>
+                            );
                         },
-                        {
-                            dataIndex: 'connectors',
-                            width: 150,
-                            title: 'Tasks',
-                            render: (_, r) => <TasksColumn observable={r} />
-                        }
-                    ]}
-                    search={{
-                        searchColumnIndex: 0,
-                        isRowMatch: (row, regex) => {
-                            const isMatch = regex.test(row.clusterName) || regex.test(row.clusterInfo.version);
-                            return isMatch;
-                        }
-                    }}
-                    observableSettings={uiSettings.kafkaConnect.clusters}
-                    pagination={{
-                        defaultPageSize: 10
-                    }}
-                    rowKey="clusterName"
-                />
-            </>
+                    },
+                    {
+                        accessorKey: 'clusterAddress',
+                        header: 'Version',
+                        cell: ({row: {original}}) => original.clusterInfo.version,
+                    },
+                    {
+                        accessorKey: 'connectors',
+                        size: 150,
+                        header: 'Connectors',
+                        cell: ({row: {original}}) => <ConnectorsColumn observable={original}/>
+                    },
+                    {
+                        accessorKey: 'connectors',
```

Comment:
I don't think this is accurate, we should use different `accessorKey` for tasks vs connectors?

Reply:
I know we had this in the code previously but just double checking this actually makes sense